ext_conf_template.txt

```diff
@@ -102,3 +102,7 @@ solr.fields.terms = terms
 solr.fields.restrictions = restrictions
 # cat=Solr; type=string; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.solr.fields.geom
 solr.fields.geom = geom
+# cat=3D Viewer; type=string; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.3dviewer.viewerModelFormatMapping
+3dviewer.viewerModelFormatMapping =
+# cat=3D Viewer; type=string; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.3dviewer.defaultViewer
+3dviewer.defaultViewer =
```

Comment:
Shouldn't there be a setting like `3dviewer.defaultViewer = model-viewer` here?

Reply:
The built-in viewer is loaded if none is set. At this point, one of the custom viewers can be configured as a fallback. I tried to make this clear in the configuration field description. Does that work for you?